### PR TITLE
Squelch repeated rkt version warnings

### PR DESCRIPTION
**Pull request by schmichael**

### CODE REVIEW

File: client/driver/rkt.go [hunk 1]
```diff
@@ -328,9 +328,10 @@ func (d *RktDriver) Fingerprint(cfg *config.Config, node *structs.Node) (bool, e
 
 	minVersion, _ := version.NewVersion(minRktVersion)
 	currentVersion, _ := version.NewVersion(node.Attributes["driver.rkt.version"])
-	if currentVersion.LessThan(minVersion) {
+	if currentVersion.LessThan(minVersion) && d.fingerprintSuccess == nil {
 		// Do not allow ancient rkt versions
-		d.logger.Printf("[WARN] driver.rkt: please upgrade rkt to a version >= %s", minVersion)
+		d.logger.Printf("[WARN] driver.rkt: unsupported rkt version %s; please upgrade to >= %s",
+			currentVersion, minVersion)
```
Comment (dadgar) [inline client/driver/rkt.go]: Should this be above the setting of the attributes and then you delete the attribute similar to how it is handle in the other failure cases?

Comment (schmichael) [reply client/driver/rkt.go]: We don't actually fail as in theory older versions will work fine with a certain subset of features.

Not sure if that's best, but it's how it has worked.

Comment (schmichael) [reply client/driver/rkt.go]: (for example all but one rkt test passes with 1.18!)

Comment (schmichael) [reply client/driver/rkt.go]: Nevermind, I'm wrong. We disable the driver. Doing Alex's suggestion.

**Pull request merged**